app/pages/authority/section/sections/capabilities/ui.test.js

```diff
@@ -0,0 +1,13 @@
+import { runTestSuite } from '../../../../../test-utils/runTestSuite';
+
+const sectionId = 'capabilities';
+
+const capabilitiesMarketingData = {
+  capabilities: 'capability stuff',
```

Comment:
let's make this '' to reflect more accurately what the API returns for the get endpoint